cloudwash/providers/ec2.py

```diff
@@ -4,8 +4,32 @@
 from cloudwash.utils import dry_data
 from cloudwash.utils import echo_dry
 from cloudwash.utils import total_running_time
+from cloudwash.utils import discovery_timestamp
 from cloudwash.client import compute_client
+import wget
+import tarfile
+import subprocess
+from os.path import exists
+import os
 
+def delete_ocp(ocp_name):
+    region = settings.providers.ec2.region
+    logger.info("Delete OCP '" + ocp_name + "'")
+    f = open("metadata.json", "w")
+    f.write("{\"clusterName\":\"" + ocp_name + "\",\"infraID\":\"" + ocp_name + "\",\"aws\":{\"region\":\"" + region + "\",\"identifier\":[{\"kubernetes.io/cluster/" + ocp_name + "\":\"owned\"}]}}")
+    f.close()
+    if exists('openshift-install'):
+        logger.info("ocp installer exists")
+    else:
+        logger.info("ocp installer doesn't exist")
+        wget.download('https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.10/openshift-install-linux.tar.gz')
+        tar = tarfile.open('openshift-install-linux.tar.gz', "r:gz")
+        tar.extractall()
+        tar.close()
+    my_env = os.environ.copy()
+    my_env["AWS_ACCESS_KEY_ID"] = settings.providers.ec2.username
+    my_env["AWS_SECRET_ACCESS_KEY"] = settings.providers.ec2.password
+    subprocess.call(['./openshift-install' , 'destroy', 'cluster', '--log-level=debug'], env=my_env)
```

Comment:
I think there should be APIs available from OCP to do this instead of using openshift-install CLI utility.

Reply:
Hi @jyejare what do you mean by "APIs available from OCP"? Unfortunately, using the openshift-install CLI tool is the only one way for uninstalling OCP I know :(  Is there any other way?